### DISCUSSION TITLE
CogVLM (agents) does poorly with text still

### DISCUSSION THREAD

**Discussion by pseudotensor** [THUDM/CogVLM#196] [Bad Case] [answered]
[fastrotate3.png.zip](https://github.com/THUDM/CogVLM/files/13687541/fastrotate3.png.zip)

![image](https://github.com/THUDM/CogVLM/assets/2249614/11d07d0d-9135-48df-bddb-8e1c7ec7055e)

**Comment by zRzRzRzRzRzRzR** [comment]
我们收到你的反馈，尝试不要使用照片，使用扫描会有更好的效果

**Comment by wenyihong** [comment, marked as answer]
hi, Thanks for your discussion. Charts and tables are relatively difficult forms of OCR, and CogAgent / VLM may fail in some cases. Thanks for bringing it up and we are actively improving our model on this. 

We've just released another checkpoint called ``cogagent-vqa``, which has better performance on single-round VQA. How about trying that?

Also, using complete sentences such as "What's the xxx of xxx?" as queries is suggested.
